pyproject.toml

```diff
@@ -5,7 +5,7 @@ build-backend = "setuptools.build_meta"
 [project]
 name = "comfystream"
 description = "Build Live AI Video with ComfyUI"
-version = "0.0.2"
+version = "0.0.3"
```

Comment:
should we also bump version in the `ui/` package file?

Reply:
Good idea, updated!